app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/event_catcher/runner.rb

```diff
@@ -14,30 +14,15 @@ def monitor_events
   end
 
   def queue_event(event)
-    _log.info("#{log_prefix} Caught event [#{event[:id]}]")
-    event_hash = event_to_hash(event, @cfg[:ems_id])
+    log_prefix = "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::EventCatcher::Runner"
```

Comment:
`log_prefix` is a common method in the base class so you don't need to override, also if you just want to print the class name that is already part of the `_log` formatter.

Here is an example from the vmware event catcher:
`_log.info("#{log_prefix} Starting Event Monitor Thread")`
yields:
`INFO -- evm: MIQ(ManageIQ::Providers::Vmware::InfraManager::EventCatcher::Runner#start_event_monitor) EMS [vc.rb.nj.grare.com] as [root] Starting Event Monitor Thread`

Reply:
Thanks! No need to override so I removed it in 2be2afa